posts/simp-made-simple.md

```diff
@@ -27,6 +27,8 @@ In this sequel, we give a more detailed exposition of the inner workings of the
 In this section we present some of the inner workings of `simp`.
 
 First we give an overview of the way `simp` works, then we delve into the specifics by introducing:
+
+
 * the `SimpM` monad, which is the metaprogramming monad holding the information relevant to a `simp` call;
```

Comment:
Ah, I meant that we want two newlines between paragraphs and bulleted lists, meaning one blank line. Sorry for the confusion! (Also below.)
```suggestion
First we give an overview of the way `simp` works, then we delve into the specifics by introducing:

* the `SimpM` monad, which is the metaprogramming monad holding the information relevant to a `simp` call;
```